test/autotest.py

```diff
@@ -1049,22 +1049,9 @@ def saveResultsNMI():
 ## runTest("timer2",   1, ["./test/timer2"])
 runTest("clock",   1, ["./test/clock"])
 
-old_ld_library_path = os.getenv("LD_LIBRARY_PATH")
-if old_ld_library_path:
-  os.environ['LD_LIBRARY_PATH'] += ':' + os.getenv("PWD") + \
-                                   "/test:" + os.getenv("PWD")
-else:
-  os.environ['LD_LIBRARY_PATH'] = os.getenv("PWD") + "/test:" + os.getenv("PWD")
-runTest("dlopen1",        1, ["./test/dlopen1"])
-# Disable the dlopen2 test until we can figure out a way to handle calls to
-# fork/exec/wait during library intialization with dlopen().
-# This seems to affect Travis CI of github, but not Ubuntu-12.04
-#if not USE_M32:
-#  runTest("dlopen2",        1, ["./test/dlopen2"])
-if old_ld_library_path:
-  os.environ['LD_LIBRARY_PATH'] = old_ld_library_path
-else:
-  del os.environ['LD_LIBRARY_PATH']
+runTest("dlopen1",       1, ["./test/dlopen1"])
```

Comment:
@karya0 and @JainTwinkle ,
    This doesn't _"break the current behavior where we are allowed to run ./autotest.py dlopen1"_.  I tested it again just now, to confirm this.
    The reason is that `autotest.py` includes a line:
```
#make sure we are in svn root 
if not os.path.isfile('./bin/dmtcp_launch'):
  os.chdir("..") 
```

So, if you start in test 'test' subdirectory, then it will detect that `./bin/dmtcp_launch` is not valid, and it will then do `os.chidr("..")`.

(But I'm change `svn` -> `github`, to bring the comment up to date.)

So, from the beginning, it could run in the github root dir, or if it is in the 'test' subdir, then it does a chdir back to the githbu root.  This allows it to use './bin/dmtcp_launch' in both cases.